modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -86,12 +86,13 @@ def new
   def create
     service_result = Storages::Storages::CreateService.new(user: current_user).call(permitted_storage_params)
     @object = service_result.result
+    @oauth_application = result_from_first_dependent_service(service_result)
```

Comment:
Even though the variable is called as what it should be, the method definitely is not. I assume a future programmer is not interested here where the `oauth_application` come from. This is already done inside the methods body. I suggest a name just like `oauth_application()`, as it suits the purpose here.